### PR TITLE
Toolbar and headline section

### PR DESCRIPTION
**Pull request by ryxtor**
This is the mobile version skeleton.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="If you need a software developer, I'm here for you, just see my work and contact me">
+        <title>Lucas' Portfolio</title>
+        <link href="style.css" rel="stylesheet">
+    </head>
+    <body>
+      <div class="container">
```
Comment (ahmetbozaci) [inline index.html]: - [x] You wrapped all content with the `div` tag. The `div` tag is unnecessary here. There is no need. For good practice, you can delete it.

Maybe you misunderstand this [practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Close%20all%20HTML,always%20needs%20%3C/div%3E). This means closing all HTML elements with closing tags.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      </li>
+      <li>
+        <a href="#"><img src="src/Angelist.svg" width="24" height="24" alt="Angelists logo"></a>
+      </li>
+      <li>
+        <a href="#"><img src="src/twitter.svg" width="24" height="24" alt="Twitter logo"></a>
+      </li>
+      <li>
+        <a href="#"><img src="src/medium.svg" width="24" height="24" alt="Medium logo"></a>
+      </li>
```
Comment (ahmetbozaci) [inline index.html]: - [x] Some icons are not showing on the page. Please fix it.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @ryxtor  ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯

- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your readme file is professional. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You stick to the design.✔️
- You know how to use flexbox very well.✔️
- You use HTML semantic elements correctly.✔️


___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [x]  Instead of using pictures for icons you can use [font-awesome-icons](https://fontawesome.com/start). 
___

### Cheers and Happy coding! 💻 👏
coding

almost


___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by ryxtor** [comment]
Hello @ahmetbozaci, thank you for the review, I did all the fixes and changed the icons images with the Awesome Fonts Icons.

**Review by julie-ify — APPROVED** [review]
## Hi @ryxtor, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**